src/main/java/com/thehrdivision/dto/EmployeeDto.java

```diff
@@ -0,0 +1,22 @@
+package com.thehrdivision.dto;
+
+import lombok.Getter;
+import lombok.Setter;
+
+import java.time.LocalDate;
+
+@Getter
+@Setter
+public class EmployeeDto {
+
+    private Integer id;
+
+    private String firstname;
+    private String lastname;
+    private String gender;
+    private LocalDate dateOfBirth;
+    private LocalDate dateJoined;
```

Comment:
Do not get it :smiley: jeżeli chodzi o nazwę of kors

Reply:
To miało oznaczać datę zatrudnienia, oraz datę zwolnienia (okfors nazwy do zmiany). Czy encja Employee to odpowiednia tabela do przechowywania tego typu danych? Mnie się wydaje, że tak.